templates/module/salt/modules/{{module_name}}.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+'''
+{{module_name}} execution module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+{{short_description}}
+
+.. versionadded:: {{version}}
+
+:configuration:
+
+    .. code-block:: yaml
+        <your example config>
+
+'''
+# keep lint from choking on _get_conn and _cache_id
+#pylint: disable=E0602
+
+from __future__ import absolute_import
+
+# Import Python libs
+import logging
+
+# Import salt libs
+import salt.utils.compat
+
+log = logging.getLogger(__name__)
+
+# Import third party libs
+HAS_LIBS = False
+try:
+    #  Import libs...
+
+    HAS_LIBS = True
+except ImportError as ie:
+    missing_package = ie.message
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = '{{module_name}}'
```

Comment:
The `__virtualname__` does not need to be the module name although it often is, perhaps add support for a configurable `virtualname` parameter defaulting to `module_name`?